components/_details.css

```diff
@@ -0,0 +1,13 @@
+.details__interessen {
```

Comment:
I'd keep it in English
```suggestion
.details__interests {
```